buildenv/Build_Instructions_V8.md

```diff
@@ -386,6 +404,48 @@ bash configure --disable-ccache \
 
 :pencil: Modify the paths for freemarker and freetype if you manually downloaded and unpacked these dependencies into different directories.
 
+2) Win 32-bit
+[1] Modify the script to support the 32-bit setting
+```
+openj9-openjdk-jdk8\jdk\make\closed\autoconf\generated-configure.sh
+  if test "x$OPENJ9_CPU" = xx86-64; then
+    if test "x$OPENJDK_BUILD_OS" = xlinux; then
+      OPENJ9_PLATFORM_CODE=xa64
+    elif test "x$OPENJDK_BUILD_OS" = xwindows; then
+      OPENJ9_PLATFORM_CODE=wa64
+      if test "x$OPENJ9_LIBS_SUBDIR" = xdefault; then
+        OPENJ9_BUILDSPEC="win_x86-64"
+      if test "x$OPENJDK_TARGET_CPU_BITS" = x32; then  <----- for 32bit setting
```

Comment:
Can you open a PR for each of the extensions repos with this change?  ie: https://github.com/ibmruntimes/openj9-openjdk-jdk8 https://github.com/ibmruntimes/openj9-openjdk-jdk9 etc

Reply:
Will do

Reply:
Making this change in all the extensions repos should be the first priority, along with setting j9vm KNOWN